File: articles/contestability-appeals-and-algorithmic-due-process/fortran/contestability_score.f90
program contestability_score
  implicit none
  real :: notice, reasons, evidence_access, human_review, correction, remedy, score
  notice = 0.70
  reasons = 0.62
  evidence_access = 0.48
  human_review = 0.55
  correction = 0.52
  remedy = 0.44
  score = (notice + reasons + evidence_access + human_review + correction + remedy) / 6.0
  print *, "contestability_score=", score
end program contestability_score

File: articles/contestability-appeals-and-algorithmic-due-process/prolog/contestability_score.pl
contestability_score(Notice, Reasons, EvidenceAccess, HumanReview, Correction, Remedy, Score) :-
    Score is (Notice + Reasons + EvidenceAccess + HumanReview + Correction + Remedy) / 6.

% Query example:
% ?- contestability_score(0.70, 0.62, 0.48, 0.55, 0.52, 0.44, Score).
